From 334cc2832770b71e5e6b6f245e5a24cced8eaac6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Sat, 25 Feb 2012 16:10:07 +0100
Subject: [PATCH] dracut-functions.sh: instmods() print only filename instead
 of fullpath

do not print the full path, when we use "find" for kernel modules, but
rather only the filename.
---
 dracut-functions.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index bd3203d..80714aa 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -1109,7 +1109,7 @@ instmods() {
                     | instmods
                 else
                     ( [[ "$_mpargs" ]] && echo $_mpargs
-                      find "$srcmods" -path "*/${_mod#=}/*" ) \
+                      find "$srcmods" -path "*/${_mod#=}/*" -printf '%f\n' ) \
                     | instmods
                 fi
                 ;;
